ocrd/ocrd/processor/base.py

```diff
@@ -95,7 +95,7 @@ def run_cli(
     log.debug("Running subprocess '%s'", ' '.join(args))
     return subprocess.call(args)
 
-class Processor(object):
+class Processor():
```

Comment:
Interesting. Is a change in some Python best practice causing the removal of the inheritance?

Reply:
It's the default in Python 3, no more old-style classes. It's a remnant of when we supported python 2.7